### PR TITLE
Mu 08

### PR DESCRIPTION
**Pull request by Staroz**
Please review Task 08, integrated cypress tests with Github Actions

image

### CODE REVIEW

File: cypress/fixtures/credentials.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "email": "",
   "pw": "",
-  "boardName": "",
-  "newBoardName": "",
+  "boardName": " ",
```
Comment (sphinix27) [inline cypress/fixtures/credentials.json]: You should remove this empty strings that just have spaces

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Currently tests are failing, they should always pass, fix the pipeline.

**Comment by Staroz** [comment]
Thanks for the recommendations, and please review the commit: 
"change Trello account get better testing and fix GitHub-Actions",
bugshttps://github.com/Staroz/Cypress_automation_training/pull/9/commits/869aa3c190b4a5435d77bf63958480f284b7b321

**Review by sphinix27 — APPROVED** [review]
LGTM!
3 out of 3 tests passed in a row.

**Pull request merged**